Calibration/HcalCalibAlgos/plugins/HcalIsoTrackAnalyzer.cc

```diff
@@ -324,6 +370,13 @@ void HcalIsoTrackAnalyzer::beginJob() {
 }
 
 // ------------ method called when starting to processes a run  ------------
+void HcalIsoTrackAnalyzer::beginRun(edm::Run const& iRun, edm::EventSetup const& iSetup) {
+  theHBHETopology_ = &iSetup.getData(tok_htopo_);
+  const HcalRespCorrs* resp = &iSetup.getData(tok_respcorr_);
+  respCorrs_ = new HcalRespCorrs(*resp);
```

Comment:
Shouldn't this get deleted at the endRun?

Reply:
Done